ffi/framebuffer_mxcfb.lua

```diff
@@ -502,6 +514,17 @@ function framebuffer:init()
         self.waveform_flashui = self.waveform_ui
         self.waveform_full = C.WAVEFORM_MODE_GC16
         self.waveform_partial = C.WAVEFORM_MODE_GC16
+    elseif self.device:isPRST() then
+        require("ffi/mxcfb_sony_h")
+
+        self.mech_refresh = refresh_prst
+        self.mech_wait_update_complete = prst_mxc_wait_for_update_complete
+
+        self.waveform_fast = C.WAVEFORM_MODE_DU
+        self.waveform_ui = C.WAVEFORM_MODE_AUTO
+        self.waveform_flashui = C.WAVEFORM_MODE_GC16
```

Comment:
In the same vein, unless there's a specific platform issue where AUTO doesn't always honor UPDATE_FULL, this should be AUTO too.